dev_guide/cron_jobs.adoc

```diff
@@ -41,26 +40,7 @@ xref:cleaning-up-after-a-cron-job[configure history limits].
 [[creating-a-cronjob]]
 == Creating a Cron Job
 
-A cron job configuration consists of the following key parts:
-
-* A schedule specified in link:https://en.wikipedia.org/wiki/Cron[cron format].
-* A job template used when creating the next job.
-* An optional deadline (in seconds) for starting the job if it misses its
-scheduled time for any reason. Missed jobs executions will be counted as failed
-ones. If not specified, there is no deadline.
-* `*ConcurrencyPolicy*`: An optional concurrency policy, specifying how to treat
-concurrent jobs within a cron job. Only one of the following concurrent
-policies may be specified. If not specified, this defaults to allowing
-concurrent executions.
-** `Allow` allows Cron Jobs to run concurrently.
-** `Forbid` forbids concurrent runs, skipping the next run if the previous has not
-finished yet.
-** `Replace` cancels the currently running job and replaces
-it with a new one.
-* An optional flag allowing the suspension of a cron job. If set to `true`,
-all subsequent executions will be suspended.
-
-The following is an example of a `*CronJob*` resource:
+The following is an example of a `CronJob` resource:
```

Comment:
Some of the styling in 3.11 is different. https://github.com/openshift/openshift-docs/blob/enterprise-3.11/contributing_to_docs/doc_guidelines.adoc#quick-markup-reference
```suggestion
The following is an example of a *CronJob* resource:
```

Reply:
Hadn't occurred to me to use a version-specific style guide.  Now I know.  Thank you.

Reply:
3.11 is the only active version that has a different guide. The one in master is valid for all of the 4.x versions. (We made some changes when we switched to the 4.x stream and didn't backport them.)